manifest/package.xml

```diff
@@ -0,0 +1,688 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
packaging data isn't needed for scratch org creation. That being said, it isn't necessarily hurting anything. I think IntelliJ/Illuminated cloud must be set to generate this in your settings. I'm going to let it through just to not keep deleting it.

Reply:
I'm cozy letting it through now, but how about (for the next release) we delete it one last time and then put it in .gitignore since it ceases to trouble us?

Probably half of the .gitignore entries in the Java projects are just us trying to smack down autogenerated IntelliJ files we don't want in the repo.